jacodb-analysis/src/main/kotlin/org/jacodb/analysis/codegen/Generator.kt

```diff
@@ -158,4 +158,13 @@ fun main(args: Array<String>) {
     }
 
     codeRepresentation.dumpTo(projectPath)
-}
\ No newline at end of file
+    try {
+        val processBuilder = ProcessBuilder()
+        processBuilder.directory(File("generated\\UtBotTemplateForIfdsSyntheticTests".replace('\\', File.separatorChar)))
+        processBuilder.command("/bin/bash", "./gradlew", "assemble")
+        processBuilder.start()
+        println("generated code built success")
```

Comment:
generation is success when return process has finished without any error exit code. You need to check it and pritn output if gradlew failed.